src/common/http/types.h

```diff
@@ -144,11 +144,12 @@ static const size_t MAX_URL_LEN = 100;
 using Url = std::array<char, MAX_URL_LEN>;
 
 // # of seconds after which nonce becomes stale for digest authentication
-// the extended version is used for requests with body, so that PrusaLink
-// hopefully never gets stale nonce for request uploading a gcode, which
-// can cause an infinit upload loop, if the browser does not read errors
-// before sending the whole body.
-static const uint32_t valid_nonce_period = 5;
-static const uint32_t extended_valid_nonce_period = 8;
```

Comment:
The `extended_valid_nonce_period` variable is no longer needed, since the only value used in now `300`.